src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -4,7 +4,7 @@ import { makeStyles } from '@material-ui/core/styles';
 import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
 import { Fab, Grid, Typography, Link } from '@material-ui/core';
 import useSiteMetadata from '../../hooks/use-site-metadata';
-import DynamicBackgroundContainer from '../DynamicBackgroundContainer';
+import DynamicBackgroundContainer from '../DynamicBackgroundContainer.jsx';
```

Comment:
We don't need `.jsx` in imports. Let's remove in this case.

Reply:
Line 7, linter was yelling about this. 🤷‍♂️.